src/cli/utils/flags.go

```diff
@@ -65,6 +65,30 @@ func AddUserFlag(cmd *cobra.Command) {
 	cobra.CheckErr(cmd.MarkFlagRequired(UserFN))
 }
 
+// AddSiteIDFlag adds the --site-id flag, which accepts site ID values.
+// This flag is hidden, since we expect users to prefer the --site url
+// and do not want to encourage confusion.
+func AddSiteIDFlag(cmd *cobra.Command) {
+	fs := cmd.Flags()
+
+	// note string ARRAY var.  IDs naturally contain commas, so we cannot accept
+	// duplicate values within a flag declaration.  ie: --site-id a,b,c does not
+	// work.  Users must call --site-id a --site-id b --site-id c.
+	fs.StringArrayVar(
+		&Site,
+		SiteIDFN, nil,
+		"Backup data by site ID; accepts '"+Wildcard+"' to select all sites.")
```

Comment:
should add a note to use the flag multiple times for multiple sites. This is a departure from how other data types work

Reply:
done